.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - "*.yml"
+      - "*.md"
+      - "*.sql"
+
+jobs:
+  lint_and_test:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Set Variables
+        id: set_variables
+        run: |
+          echo "::set-output name=PY::$(python -c 'import hashlib, sys;print(hashlib.sha256(sys.version.encode()+sys.executable.encode()).hexdigest())')"
+          echo "::set-output name=PIP_CACHE::$(pip cache dir)"
+      - name: Cache PIP
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.set_variables.outputs.PIP_CACHE }}
+          key: ${{ runner.os }}-pip-${{ steps.set_variables.outputs.PY }}
+      - name: Install dependencies
+        run:  |
+          python -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
+      - name: Run black lint
+        run:  | 
+          black .  --line-length 100
```

Comment:
Even with the cfg file, we still need to pass the argument here?

Reply:
yes...for command line `black` needs it.

Reply:
no need, (wo shabi le)

Reply:
fixed.....